pkgs/development/python-modules/nose/default.nix

```diff
@@ -26,7 +26,7 @@ buildPythonPackage rec {
   preBuild = lib.optionalString
       ((python.isPy3k or false) && (python.pname != "pypy3"))
   ''
-    2to3 -wn nose functional_tests unit_tests
+    ${python.pythonForBuild}/bin/2to3 -wn nose functional_tests unit_tests
```

Comment:
Wasn't 2to3 removed or was this only in setuptools?

Reply:
2to3 is definitely needed here, and still included in python3, if that's what you mean. This PR actually fixes the nose cross build for me.